.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
Please remove the .gitignore file from this pull request.
I want this pull request to focus only on the looks of the web page